pom.xml

```diff
@@ -43,7 +43,43 @@
 		<maven.compiler.target>1.8</maven.compiler.target>
 	</properties>
 
-
+	<profiles>
+		<profile>
+			<id>release-sign-artifacts</id>
+			<activation>
+				<property>
+					<name>performRelease</name>
+					<value>true</value>
```

Comment:
CI was mainly broken because GPG signing was active (smart) but the CI bot doesn't have the GPG keys (also smart).

this stanza moves the existing GPG configuration into a "profile," wherein, GPG signing and other plugins meant to be active during a release can be "turned on," so to speak, by passing `-DperformRelease` (defining a particular system property).

so, if you want to activate GPG signing, you just do this:
`mvn clean package install -DperformRelease`

this is detailed in the following SO article but the approach is common:
https://stackoverflow.com/questions/14825039/suppressing-gpg-signing-for-maven-based-continuous-integration-builds-travis-ci

obviously because this changes the workflow for bigchain's library, it is entirely up to you whether you like this and want to merge it. but it does fix CI.